templates/admin/asset/image_editor.html.twig

```diff
@@ -127,13 +142,47 @@
             return false;
         });
     }
+
+    function manipulateMenuBar() {
+        const menuBar = document.getElementsByClassName('menu_bar')[0];
+        const liElement = document.createElement('li');
+        const aElement = document.createElement('a');
+        const spanElement = document.createElement('span');
+
+        aElement.setAttribute('id', 'save_button');
+        spanElement.setAttribute('class', 'name trn');
+        spanElement.innerText = parent.t('save');
+
+        aElement.appendChild(spanElement);
+        liElement.appendChild(aElement);
+        menuBar.prepend(liElement);
+
+        document.getElementById('main_menu_0_0').addEventListener('click', () => {
+            document.getElementById('main_menu_1_10').remove();
+            document.getElementById('main_menu_1_11').remove();
+        })
+
+        removeHref();
+
+        document.getElementsByTagName('nav')[0].addEventListener('click', (e) => {
+            removeHref();
+        });
+    }
+
+    function removeHref(className = 'main_menu') {
+        const allLinkElements = document.getElementsByTagName('nav')[0].querySelectorAll("a");
+        for (let a of allLinkElements) {
+            if (a.getAttribute('href') === 'javascript:void(0)') {
+                a.removeAttribute('href');
+            }
+        }
+    }
     
     window.addEventListener("load", function(e) {
```

Comment:
Do we need this at all?

Reply:
Yes we call there the function loadEditor

Reply:
Okay, but the `.then()` part with the `console.log`?

Reply:
I think we can remove that